### PR TITLE
@eessex => nowrap for single artist names; wrap for multiple

### PR DESCRIPTION
**Pull request by sarahscott**
Before:

screen shot 2018-03-20 at 1 09 53 pm

After:
screen shot 2018-03-21 at 12 28 20 pm

Fixes https://artsyproduct.atlassian.net/browse/GROWTH-254

### CODE REVIEW

File: src/Components/Publishing/Sections/ArtworkCaption.tsx [hunk 1]
```diff
@@ -39,7 +43,7 @@ export class ArtworkCaption extends React.Component<ArtworkCaptionProps, null> {
     const { artwork: { artist, artists } } = this.props
 
     // Multiple artists
-    if (artists && artists.length > 0) {
```
Comment (sarahscott) [inline src/Components/Publishing/Sections/ArtworkCaption.tsx]: The `artists` array contains an `artist` even if there is just one.

**Comment by sarahscott** [comment]
Actually it might make sense to do 'nowrap' on the individual artist names instead of the whole string.. will try that now.

File: src/Components/Publishing/Sections/ArtworkCaption.tsx (hunk 1 above)
Comment (eessex) [reply src/Components/Publishing/Sections/ArtworkCaption.tsx]: This conditional could be a const, something list `hasMultipleArtists` -- noticed you use it again on 207

File: src/Components/Publishing/Sections/ArtworkCaption.tsx [hunk 2]
```diff
@@ -196,11 +204,11 @@ export class ArtworkCaption extends React.Component<ArtworkCaptionProps, null> {
           sectionLayout={sectionLayout}
           className="display-artwork__caption"
         >
-          <ArtistName>{this.renderArtists()}</ArtistName>
-
+          <ArtistNames multipleNames={artists && artists.length > 1}>
+            {this.renderArtists()}
```
Comment (eessex) [inline src/Components/Publishing/Sections/ArtworkCaption.tsx]: Related to above, names is not super-specific, I think `hasMultipleArtists` is more explicit about the expectations

**Comment by sarahscott** [comment]
I figured out a way to make the individual names `nowrap` instead of all of the names together and I think it looks better during resizing (bc the whole name will go to the next line instead of splitting up last/first names).

**Pull request merged**